bin/pa11y-scan

```diff
@@ -6,25 +6,5 @@
 #  * postgresql running
 #  * no other server is listening on port 3000
 
-export RAILS_ENV=ci
 
-# ensure assets are properly compiled for CI environment
-bundle exec rake assets:clobber
-bundle exec rake assets:precompile
-
-# run the server
-bundle exec rails server &
-server_pid=$!
-# pause to ensure the server has started
-sleep 5
-
-# ensure pa11y-ci is installed
-yarn install --frozen-lockfile
-
-yarn run pa11y-ci
-exit_status=$?
-
-# shut down the server
-kill $server_pid
-
-exit $exit_status
+`dirname "$0"`/with-server "yarn run pa11y-ci"
```

Comment:
this is great. do you have other uses of `with-server` in mind?

Reply:
Not immediately, beyond pa11y and owasp.